qemu/tests/cfg/qemu_img_measure.cfg

```diff
@@ -2,6 +2,7 @@
     virt_test_type = qemu
     kill_vm = yes
     start_vm = no
+    required_qemu = (2.10.0, )
```

Comment:
Measure has been supported with below patch in Upstream QEMU since v2.10.0-rc0.
commit fd03c2b8fbf463f92413ed70a978bf70ebbc74c7
Author: Stefan Hajnoczi <stefanha@redhat.com>
Date:   Wed Jul 5 13:57:36 2017 +0100

    qemu-img: add measure subcommand
    
    The measure subcommand calculates the size required by a new image file.
    This can be used by users or management tools that need to allocate
    space on an LVM volume, SAN LUN, etc before creating or converting an
    image file.
    
    Suggested-by: Maor Lipchuk <mlipchuk@redhat.com>
    Signed-off-by: Stefan Hajnoczi <stefanha@redhat.com>
    Reviewed-by: Alberto Garcia <berto@igalia.com>
    Message-id: 20170705125738.8777-8-stefanha@redhat.com
    Signed-off-by: Max Reitz <mreitz@redhat.com>

Reply:
Updated. Thanks.